system_tests/tests/tests.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.test_modes import TestModes
+from utils.testing import get_running_lewis_and_ioc
+from parameterized import parameterized
+
+DEVICE_PREFIX = "HAMEG8123_01"
+
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("HAMEG8123"),
+        "macros": {},
+        "emulator": "hameg_8123",
+    },
+]
+
+
+TEST_MODES = [TestModes.DEVSIM]
```

Comment:
Was it intentional to not run *any* tests in recsim?

Reply:
nah i'm just lazy!